server/test/framework/Library_Test.cpp

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) Huawei Technologies Co., Ltd. 2012-2021. All rights reserved.
+ */
+
+#include "gtest/gtest.h"
+
+#include "BaseTest.h"
+
+namespace {
+    class Library_Test : public BaseTest {
+    protected:
+        Library_Test() : BaseTest("library-project") {
+        }
+
+        fs::path test = getTestFilePath("src");
+        fs::path sum = getTestFilePath("lib");
+        fs::path test_c = test / "test.c";
+
+        void SetUp() override {
+            clearEnv();
+            srcPaths = { suitePath, test, sum };
+        }
+
+        std::pair<FunctionTestGen, Status>
+        createTestForFunction(const fs::path &pathToFile, int lineNum, int kleeTimeout = 60) {
+            auto lineRequest = testUtils::createLineRequest(projectName, suitePath, buildDirRelativePath,
+                                                 srcPaths, pathToFile, lineNum, false, kleeTimeout);
+            auto request = GrpcUtils::createFunctionRequest(std::move(lineRequest));
+            auto testGen = FunctionTestGen(*request, writer.get(), TESTMODE);
+            testGen.setTargetForSource(pathToFile);
+            Status status =
+                Server::TestsGenServiceImpl::ProcessBaseTestRequest(testGen, writer.get());
+            return { testGen, status };
+        }
+    };
+
+    TEST_F(Library_Test, sum) {
+        auto [testGen, status] = createTestForFunction(test_c, 7);

```

Comment:
Add one more call `createTestForFunction(test_c, 7);` in same test

Reply:
Done